docs/FAQ.md

```diff
@@ -0,0 +1,283 @@
+# FAQ: Frequently Asked Questions about YourBench
```

Comment:
Why is this not in the README/linked from the README?

Reply:
(good doc overall)

Reply:
I was planning to edit the readme next (to show pip install instructions as well as link the FAQ once we release the package :) )